.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "python.formatting.provider": "black",
+  "python.formatting.blackArgs": [
+    "-l",
+    "120"
+  ],
+  "editor.formatOnSave": true
+}
```

Comment:
Are you sure you need to push this file to the repo?

Reply:
Yes! It provides automatic formatting for my editor. It's the same code formatter that we use in our CI, so it makes sense that we would like to run it every now and then rather than fix everything manually. I might agree this is not the best PR for it though.